.gitignore

```diff
@@ -0,0 +1 @@
+*test.md
```

Comment:
- [ ] Also you should have added the linters so you can add the `node_modules` to the `gitignore` file [click here to add linters](https://github.com/microverseinc/linters-config/tree/master/html-css).

![MMM](https://user-images.githubusercontent.com/61562626/137922392-d7fa15ed-e779-4ee9-b355-f3fbcc984114.jpeg)